high_card.rb

```diff
@@ -3,3 +3,41 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+full_suits = []
+full_deck = []
+input = ""
+player = ""
+players = []
+#
+# puts ranks + "all ranks"
+# puts suits + "all suits"
+# puts "all card types noted"
+
+ace = ranks.shift[0]
+ordered_ranks = ranks.push(ace)
+puts ordered_ranks
+
+ordered_ranks.each do |rank|
+  suits.each do |suit|
+    full_deck << [rank, suit]
+  end
+end
+#
+# full_deck = ordered_ranks.product(suits)
+puts full_deck.inspect
+
+shuffled_deck = full_deck.shuffle
+puts shuffled_deck.inspect
+#
+# while player != "play"
+#   puts "input player name"
+#   player = gets.chomp.to_s
+#   if player == "play"
+#     break
```

Comment:
i don't think you'll need this because once the player becomes play it will exit the loop anyway.